packages/cojson-transport-ws/src/index.ts

```diff
@@ -106,3 +125,15 @@ export function createWebSocketPeer(options: {
         };
     });
 }
+
+const addEventListener = <Event extends keyof WebsocketEvents, R>(
```

Comment:
technically I think it's better to make this a `acquireRelease` because if we register an event listener we should also free it when a Scope is closed. wdyt?

Reply:
this operator is redundant, we can replace it wholesale with `Stream.fromEventListener`